src/components/ui/SkuSelector/SkuSelector.tsx

```diff
@@ -85,7 +87,7 @@ function SkuSelector({
     <div data-store-sku-selector data-testid={testId} data-variant={variant}>
       {label && (
         <Label data-sku-selector-label>
-          {label}: {selectedSku}
+          {label}: <strong>{selectedSku}</strong>
```

Comment:
Unless we are judging that the text between the `<strong>` tag is so important, we should keep it. But if it is just for style purposes, I think we can use `font-weight`?

ref: https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/strong#notas_de_uso

Reply:
It's more important for colors, for accessibility proposes